libraries/azure-app-configuration-importer/src/featureFlag.ts

```diff
@@ -0,0 +1,61 @@
+import { ClientFilter } from "./models";
+import { FeatureFlagValue as FeatureFlagValue1 } from "@azure/app-configuration";
```

Comment:
Use it's origin name, don't use alias for this type

Reply:
Updated